docs/Queries.md

```diff
@@ -193,6 +218,41 @@ const deleteAction = api.del(deleteQuery);  // note `api.del` not `api.delete` b
 Use Redux's `store.dispatch` or `bindActionCreators` to dispatch the query
 actions.
 
+#### `Promise` interface for dispatched queries
+
+An API request action object always has a type of `API_REQ`, and it will always
+have a `meta` property that contains a `request` property corresponding to the
+current status of the request. The Promise will resolve on a successful API
+request, and `reject` on a 400/500 error from the fetch call.
+
+This property can be accessed by the `dispatch` _caller_ by consuming the return
+value of the `dispatch()` call, which is the dispatched action itself. The
+caller can then attach response handlers in `action.meta.request.then()` Promise
+callbacks.
+
+**Example using `mapDispatchToProps`**
+
+```js
+// Example.jsx
+import { SubmissionError } from 'redux-form';
+import * as api from 'meetup-web-platform/lib/actions/apiActionCreators';
+
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators({ get: api.get }, dispatch);
+}
+
+class Example extends React.Component {
+  onSubmit() {
+    const formQuery = { ... };
+    const apiRequest = api.post(formQuery); // this triggers the POST
```

Comment:
Just trying to follow along and was wondering if this should be calling something like `this.props.get` instead of `api.post` in your example?